corehq/apps/userreports/tasks.py

```diff
@@ -260,7 +255,7 @@ def _iteratively_build_table(config, resume_helper=None, in_place=False, limit=-
 
 @task(serializer='pickle', queue=UCR_CELERY_QUEUE, ignore_result=True)
 def delete_data_source_task(domain, config_id):
-    from corehq.apps.userreports.views import delete_data_source_shared
+    from .views import delete_data_source_shared
```

Comment:
I didn't know that isort was configured or itself prefers the dot notation for modules within the same app.